agent_setup.sh

```diff
@@ -87,6 +88,10 @@ function initialize_input(){
                 is_install_and_config="false"
                 shift 1
                 ;;
+            --container-agent)
```

Comment:
I'm ok with this. But more elegantly we could have a different way for the script to understand if the environment is running is a container. But for now this should work just fine.